### ISSUE TITLE
Possibility to use dropout in vgg16/19

### ISSUE THREAD

**Issue by zaccharieramzi** [keras-team/keras#16550]
If you open a GitHub issue, here is our policy:

It must be a bug, a feature request, or a significant problem with the documentation (for small docs fixes please send a PR instead).
The form below must be filled out.

**Here's why we have that policy:**.

Keras developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

**System information**.

TensorFlow version (you are using): 2.8
Are you willing to contribute it (Yes/No) : Yes

**Describe the feature and the current behavior/state**.

I would like to be able to use dropout on the first two fully connected layers of the VGG implementations in keras applications, as specified [in the original paper](https://arxiv.org/abs/1409.1556) (and [done in PyTorch](https://github.com/pytorch/vision/blob/main/torchvision/models/vgg.py#L46)).

> dropout regularisation for the first two fully-connected layers (dropout ratio set to 0.5)

**Will this change the current api? How?**

Yes: this will mean adding a `dropout` kwarg to the VGG implementations, that will default to `0.0` to remain a non-breaking change.

**Who will benefit from this feature?**

Users willing to reproduce the original results of the VGG paper.

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**

- Do you want to contribute a PR? (yes/no): yes
- If yes, please read [this page](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md) for instructions
- Briefly describe your candidate solution(if contributing): just adding dropout layers.

**Comment by qlzh727** [comment]
Ack, just as reference, the original paper stated that

"The training was regularised by weight decay (the L2 penalty multiplier set to
5 · 10−4) and dropout regularisation for the first two fully-connected layers (dropout ratio set to 0.5)."

**Comment by qlzh727** [comment]
Feel free to send any PR if you would like to contribute. Thanks

**Comment by zaccharieramzi** [comment]
Closing as per [this](https://github.com/keras-team/keras/pull/16631#issuecomment-1144356554).
